packages/yoastseo/spec/languageProcessing/languages/en/ResearcherSpec.js

```diff
@@ -83,4 +76,9 @@ describe( "a test for the English Researcher", function() {
 		};
 		expect( researcher.getHelper( "fleschReadingScore" )( statistics ) ).toBe( 17.3 );
 	} );
+
+	it( "checks is a word is complex in English", function() {
```

Comment:
If I understand correctly, `investigations` would be complex before this PR right?

Reply:
Yes, this word was marked as complex before